client/me/help/style.scss

```diff
@@ -46,3 +46,38 @@
 	font-size: 14px;
 	color: $gray-dark;
 }
+
+.help__course-teaser .card {
```

Comment:
@mtias This gets a bit complicated as I have to do something similar but using a different highlight color.
In that case would you do something like `<Card highlighted="orange" />` or just `<Card highlighted/>` and then add css to make it orange.